proto/proto/store.proto

```diff
@@ -23,6 +23,11 @@ service Api {
     // Returns the latest state of an account with the specified ID.
     rpc GetAccountDetails(requests.GetAccountDetailsRequest) returns (responses.GetAccountDetailsResponse) {}
 
+    
+    // Returns the latest state of an account with the specified ID.
+    // TODO: would probably be cleaner to do a specific response type instead of reusing GetAccountDetailsResponse
+    rpc GetNetworkAccountDetailsByPrefix(requests.GetNetworkAccountDetailsByPrefixRequest) returns (responses.GetAccountDetailsResponse) {}
```

Comment:
Agreed re specific response type - I'd add this to the list of follow-ups for future PRs.

Reply:
Ended up doing it here: https://github.com/0xMiden/miden-node/pull/840/commits/5069409e9b47678d5a6cd385b384600abd4c251d